<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ogcalc</class>
 <widget class="QMainWindow" name="ogcalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>123</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OG &amp; ABV calculator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../ogcalc/ogcalc.svg</normaloff>../../../../../ogcalc/ogcalc.svg</iconset>
  </property>
  <widget class="QWidget" name="toplevel_layout">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="vertical_layout">
    <item>
     <layout class="QHBoxLayout" name="entry_layout">
      <item>
       <layout class="QHBoxLayout" name="pg_layout">
        <item>
         <widget class="QLabel" name="pg_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Present Gravity (density)</string>
          </property>
          <property name="text">
           <string>&amp;PG:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="buddy">
           <cstring>pg_entry</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="SpinBoxEntry" name="pg_entry">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="ri_layout">
        <item>
         <widget class="QLabel" name="ri_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Refractive Index</string>
          </property>
          <property name="text">
           <string>R&amp;I:</string>
          </property>
          <property name="buddy">
           <cstring>ri_entry</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="SpinBoxEntry" name="ri_entry">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="cf_layout">
        <item>
         <widget class="QLabel" name="cf_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Correction Factor</string>
          </property>
          <property name="text">
           <string>C&amp;F:</string>
          </property>
          <property name="buddy">
           <cstring>cf_entry</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="SpinBoxEntry" name="cf_entry">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="result_layout">
      <item>
       <layout class="QHBoxLayout" name="og_layout">
        <item>
         <widget class="QLabel" name="og_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Original Gravity (density)</string>
          </property>
          <property name="text">
           <string>OG:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="og_result">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="abv_layout">
        <item>
         <widget class="QLabel" name="abv_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Percent Alcohol By Volume</string>
          </property>
          <property name="text">
           <string>ABV %:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="abv_result">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="button_layout">
      <item>
       <widget class="QPushButton" name="quit">
        <property name="text">
         <string>&amp;Quit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reset">
        <property name="text">
         <string>&amp;Reset</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="calculate">
        <property name="text">
         <string>&amp;Calculate</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SpinBoxEntry</class>
   <extends>QDoubleSpinBox</extends>
   <header>SpinBoxEntry.h</header>
   <slots>
    <signal>enterPressed()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>reset</sender>
   <signal>clicked()</signal>
   <receiver>ogcalc</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quit</sender>
   <signal>clicked()</signal>
   <receiver>ogcalc</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calculate</sender>
   <signal>clicked()</signal>
   <receiver>ogcalc</receiver>
   <slot>calculate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calculate</sender>
   <signal>clicked()</signal>
   <receiver>reset</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pg_entry</sender>
   <signal>enterPressed()</signal>
   <receiver>ri_entry</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ri_entry</sender>
   <signal>enterPressed()</signal>
   <receiver>cf_entry</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cf_entry</sender>
   <signal>enterPressed()</signal>
   <receiver>calculate</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pg_entry</sender>
   <signal>enterPressed()</signal>
   <receiver>ri_entry</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ri_entry</sender>
   <signal>enterPressed()</signal>
   <receiver>cf_entry</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>calculate()</slot>
  <slot>reset()</slot>
 </slots>
</ui>
